defaults/main.yml

```diff
@@ -1,11 +1,32 @@
-# Dont change this value, it's value is referenced
+
+# Tarsnap key contents
+# Should look like
+# TARSNAP_KEY: |
+#      # START OF TARSNAP KEY FILE
+#      contents
+#      # END OF TARSNAP KEY FILE
+TARSNAP_KEY: null
 TARSNAP_KEY_REMOTE_LOCATION: /root/tarsnap.key
-TARSNAP_KEY_LOCAL_TEMPLATE: null
 
-TARSNAPPER_CONF: null
+# You'll need to upload these scripts by yourself
+TARSNAP_BACKUP_PRE_SCRIPT: null
+TARSNAP_BACKUP_POST_SCRIPT: null
+
```

Comment:
Perfect - for the config files I had in mind the tarsnaper config, but it's true that you don't use that here. Failing if the variables aren't defined works well for me.